docs/pages/api/app-bar.md

```diff
@@ -9,7 +9,7 @@ filename: /packages/material-ui/src/AppBar/AppBar.js
 <p class="description">The API documentation of the AppBar React component. Learn more about the properties and the CSS customization points.</p>
 
 ```js
-import AppBar from '@material-ui/core/AppBar';
+import { AppBar } from '@material-ui/core';
```

Comment:
@oliviertassinari  Again the reminder to use descriptive PR titles. I didn't look at this PR because it didn't concern most of the things I work on. But it does contradict our current recommended import and our docs style. If you have a rationale for changing it please be so kind and share it with the rest of the team.

Reply:
This was a preliminary step to trigger discussions like #16803, I'm happy with the outcome.

Reply:
> I'm happy with the outcome. 

And that stands above all. Glad you're fine with reverting the work of others without discussion. Good for you.